From 3ae06e10e5e25ca6aab04eba1cb0402bfe068997 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Tue, 5 Mar 2019 17:23:24 +0100
Subject: [PATCH 10/63] ucs2.h: remove unused variable

The const uint16_t pointer is not used since now the two bytes of the
UCS-2 chars are checked to know if is the termination of the string.

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 src/ucs2.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/ucs2.h b/src/ucs2.h
index edd8367b4bc..e0390c34985 100644
--- a/src/ucs2.h
+++ b/src/ucs2.h
@@ -26,12 +26,11 @@ static inline size_t UNUSED
 ucs2len(const void *vs, ssize_t limit)
 {
 	ssize_t i;
-	const uint16_t *s = vs;
 	const uint8_t *s8 = vs;
 
 	for (i = 0;
 	     i < (limit >= 0 ? limit : i+1) && s8[0] != 0 && s8[1] != 0;
-	     i++, s8 += 2, s++)
+	     i++, s8 += 2)
 		;
 	return i;
 }
-- 
2.26.2

